get-started/package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Basic example of a UI Extensions built with HubSpot's React developer tools",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "postinstall": "cd ./src/app/extensions/ && npm install",
+    "build": "npm run build --prefix ./src/app/extensions/",
+    "dev": "npm run dev --prefix ./src/app/extensions/"
```

Comment:
Can we do this for the other two projects as well?

Reply:
@dshekhtman yup, definitely. I just wanted to get an overall sentiment on this first before doing that.

I will update those now 👍

Reply:
Updated the other two projects: ab1db63c1ffccc202fec2466e1fd8c1108b61459